speech.js

```diff
@@ -21,6 +21,8 @@ class Speech {
     formData.append('text', text);
     formData.append('voice', voice);
     formData.append('seed', options.seed || 0);
```

Comment:
we should let these take the default values on the server side if not specified to give us flexibility to adjust without updating sdks

e.g. no seed means random, but here we're fixing it to 0 all the time

Reply:
Done.